types/lodash/lodash-tests.ts

```diff
@@ -4495,6 +4495,12 @@ fp.now(); // $ExpectType number
     _(42).isObjectLike(); // $ExpectType boolean
     _.chain([]).isObjectLike(); // $ExpectType PrimitiveChain<boolean>
     fp.isObjectLike(anything); // $ExpectType boolean
+    if (fp.isObjectLike(anything)) {
```

Comment:
I think this test is failing. You'll probably need to manually update the `fp` version (in `fp.d.ts` IIRC).